salt/modules/saltutil.py

```diff
@@ -1273,7 +1273,7 @@ def _exec(client, tgt, fun, arg, timeout, tgt_type, ret, kwarg, **kwargs):
         _cmd = client.cmd_subset
         cmd_kwargs = {
             'tgt': tgt, 'fun': fun, 'arg': arg, 'tgt_type': tgt_type,
-            'ret': ret, 'kwarg': kwarg, 'sub': kwargs['subset'],
+            'ret': ret, 'cli': True, 'kwarg': kwarg, 'sub': kwargs['subset'],
```

Comment:
Is this change supposed to be included here?

Reply:
Yes, i just wanted to put it in the list before kwarg.

This is the portion that returns everything in the correct format so that it can be processed by saltmod.

I thought that I had done this when I originally submitted the cmd_subset stuff, but I apparently had not.